usecases/config/environment.go

```diff
@@ -442,13 +449,13 @@ const (
 )
 
 const (
-	DefaultPersistenceFlushIdleMemtablesAfter = 60
-	DefaultPersistenceMemtablesMaxSize        = 200
-	DefaultPersistenceMemtablesMinDuration    = 15
-	DefaultPersistenceMemtablesMaxDuration    = 45
-	DefaultMaxConcurrentGetRequests           = 0
-	DefaultGRPCPort                           = 50051
-	DefaultMinimumReplicationFactor           = 1
+	DefaultPersistenceMemtablesFlushDirtyAfter = 60
```

Comment:
Keeping the same threshold will result in a lot more flushes on batch inserts; this is a good thing and closer what a user would expect. I'll do some benchmarking in this branch to see if there are performance implications.

Reply:
Great! Thank you.